src/ValidatorKeys.h

```diff
@@ -79,14 +89,11 @@ class ValidatorKeys
     ValidatorKeys(ValidatorKeys const&) = default;
     ValidatorKeys& operator=(ValidatorKeys const&) = default;
 
-    inline bool
-    operator==(ValidatorKeys const& rhs) const
-    {
-        // TODO Compare secretKey_
-        return revoked_ == rhs.revoked_ &&
-            keyType_ == rhs.keyType_ &&
-            tokenSequence_ == rhs.tokenSequence_ &&
-            publicKey_ == rhs.publicKey_;
+    inline bool operator==(ValidatorKeys const &rhs) const {
+      return revoked_ == rhs.revoked_ && keyType_ == rhs.keyType_ &&
+             tokenSequence_ == rhs.tokenSequence_ &&
+             keys_.publicKey == rhs.keys_.publicKey &&
+             keys_.secretKey == rhs.keys_.secretKey;
```

Comment:
Nit: Formatting. The `return` needs two more spaces. And the other conditions probably will, too.